src/ui/molecules/pricing-table.svelte

```diff
@@ -163,6 +163,10 @@
     color: var(--rc-color-grey-text-dark);
   }
 
+  .rcb-pricing-table-value {
+    text-align: right;
```

Comment:
This seems unrelated. Did you find a style issue in the pricing table?

Reply:
I have no idea how this got here 🤔 . Thanks for the heads up.